### PR TITLE
[2.6] Fix HTML escaping of to-source links

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #14694 |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

File: src/Symfony/Component/Debug/ExceptionHandler.php [hunk 1]
```diff
@@ -380,7 +380,7 @@ private function formatPath($path, $line)
         $file = preg_match('#[^/\\\\]*$#', $path, $file) ? $file[0] : $path;
 
         if ($linkFormat = $this->fileLinkFormat) {
-            $link = str_replace(array('%f', '%l'), array($path, $line), $linkFormat);
+            $link = strtr($this->escapeHtml($linkFormat), array('%f' => $path, , '%l' => (int) $line));
```
Comment (burci) [inline src/Symfony/Component/Debug/ExceptionHandler.php]: This line introduces an error, too much comma...

**Comment by burci** [comment]
@nicolas-grekas can you change the two commas to one? Or should i make a pull request?

File: src/Symfony/Component/Debug/ExceptionHandler.php (hunk 1 above)
Comment (fabpot) [inline src/Symfony/Component/Debug/ExceptionHandler.php]: fixed in 1e06857

**Comment by amenk** [comment]
Do you know when https://github.com/symfony/debug will be updated?

**Pull request merged**